snet/cli/config.py

```diff
@@ -203,19 +218,51 @@ def _persist(self):
             self.write(f)
         self._config_file.chmod(0o600)
 
-
-def first_identity_message_and_exit():
-    print("\nPlease create your first identity by running 'snet identity create'.\n\n"
-          "The available identity types are:\n"
-          "    - 'rpc' (yields to a required ethereum json-rpc endpoint for signing using a given wallet\n"
-          "          index)\n"
-          "    - 'mnemonic' (uses a required bip39 mnemonic for HDWallet/account derivation and signing\n"
-          "          using a given wallet index)\n"
-          "    - 'key' (uses a required hex-encoded private key for signing)\n"
-          "    - 'ledger' (yields to a required ledger nano s device for signing using a given wallet\n"
-          "          index)\n"
-          "    - 'trezor' (yields to a required trezor device for signing using a given wallet index)\n"
-          "\n")
+    def get_param_from_sdk_config(self, param: str, alternative=None):
+        if self.sdk_config:
+            return self.sdk_config.get(param, alternative)
+        return None
+
+    def setup_identity(self):
+        identity_type = self.get_param_from_sdk_config("identity_type")
+        private_key = self.get_param_from_sdk_config("private_key")
+        default_wallet_index = self.get_param_from_sdk_config("wallet_index", 0)
+        if not identity_type:
+            raise Exception("identity_type not passed")
+        if identity_type == "key":
+            identity = {
+                "identity_type": "key",
+                "private_key": private_key,
+                "default_wallet_index": default_wallet_index
+            }
+        # TODO: logic for other identity_type
+        else:
+            print("\nThe identity_type parameter value you passed is not supported "
+              "by the sdk at this time.\n")
+            print("The available identity types are:\n"
+                  "    - 'key' (uses a required hex-encoded private key for signing)\n\n")
+            exit(1)
+        return identity
+
+
+def first_identity_message_and_exit(is_sdk=False):
+    if is_sdk:
+        print("\nPlease create your first identity by passing the 'identity_name' "
+              "and 'identity_type' parameters in the config.\n")
```

Comment:
When displaying this message, **need to clearly highlight that you need to fill in new fields in the config, which refers to the SDK**